packages/mdc-select/index.js

```diff
@@ -145,7 +145,7 @@ class MDCSelect extends MDCComponent {
       this.outline_ = outlineFactory(outlineElement);
     }
 
-    if (this.root_.classList.contains(cssClasses.BOX)) {
+    if (!this.root_.classList.contains(cssClasses.OUTLINED)) {
```

Comment:
This seems like a change of behavior. Did outlined previously not have ripples?

Reply:
Never mind... I missed the `!`.

Reply:
The outlined variant does not have ripples. This code used to single out the box variant to add ripples, but since there is no longer a default variant this needs to exclude the outlined variant.